### PR TITLE
add multi-config support example

### PR DESCRIPTION
**Pull request by Galaoaoa**
* Add multi-config support example

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,6 +12,7 @@ We want examples of as many use cases in this repository as possible! Submit a P
 ### Basic FireLens examples
 * [Using the 'file' 'config-file-type'](examples/fluent-bit/config-file-type-file)
 * [Cross Account Log Forwarding](examples/fluent-bit/cross-account)
+* [Multiple Config Support](examples/fluent-bit/multi-config-support)
```
Comment (PettitWesley) [inline README.md]: Same comment as on your use case guide, let's make this title better and longer to advertise teh feature. 

Source multiple configs from S3 or files

Or a better title if you can think of one.

File: examples/fluent-bit/multi-config-support/README.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    bucket                       your-bucket
+    region                       ${AWS_REGION}
+    total_file_size              1M
+    upload_timeout               1m
+    use_put_object               On
+```
+
+
+
+**Note:** you can find these config files in the `config-files` directory of this example, please modify them according to the actual situation to match your log file.
```
Comment (PettitWesley) [inline examples/fluent-bit/multi-config-support/README.md]: "to match your needs" would be better I think

File: examples/fluent-bit/multi-config-support/README.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
+## FireLens Example: Multiple Config support - using the Fluent Bit image with init tag
+
+This example shows you how to set multiple config files for Fluent Bit on ECS.
```
Comment (PettitWesley) [inline examples/fluent-bit/multi-config-support/README.md]: Let's link the use case guide here as well and say it explains how and why init was created

Comment (Galaoaoa) [reply examples/fluent-bit/multi-config-support/README.md]: Hmm maybe I need to merge use case guide first, get the real link and then add the link in it..
Actually linking them to each other..

Comment (PettitWesley) [reply examples/fluent-bit/multi-config-support/README.md]: You can predict what the link will be based on the existing links and your new file name

File: examples/fluent-bit/multi-config-support/task-definition.json [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+			"logConfiguration": {
+				"logDriver": "awslogs",
+				"options": {
+					"awslogs-group": "firelens-container",
+					"awslogs-region": "us-west-2",
+					"awslogs-create-group": "true",
+					"awslogs-stream-prefix": "firelens-init"
+				}
+			},
+            "environment": [
```
Comment (PettitWesley) [inline examples/fluent-bit/multi-config-support/task-definition.json:28]: the JSON formatting is a bit messed up, run this through something like this to make it pretty: https://jsonformatter.org/json-pretty-print

**Review by PettitWesley — APPROVED** [review]

File: examples/fluent-bit/multi-config-support/task-definition.json (hunk 4 above)
Comment (Galaoaoa) [reply examples/fluent-bit/multi-config-support/task-definition.json:28]: That's a great tool, the format now pretty standard!

**Review by PettitWesley — APPROVED** [review]

**Pull request merged**